Invert tagging invocation logic

mijicd opened the pull request:
Using tags, instead of skip-tags to invoke the appropriate tasks. 

Fixed the SSH timeout error while starting the existing instances. The problem was caused by the fact that stopped instances does not have a public DNS name. It is solved by acquiring the remote facts immediately after starting them.

mijicd committed:
Fix SSH timeout when starting AWS instances

Since stopped instance do not have a public DNS name, the SSH
connections to them could not be established. The problem is solved by
acquiring the remote facts after the instances are started.

mgobec commented on projects/anomaly-detection-stack/layers/monitoring.yml:
```diff
@@ -25,5 +25,4 @@
     - riemann-docker
-  tags:
-    - aws-start
+  tags: ['aws-setup', 'aws-start']
```
Why this task has two tags and the top one is not tagged? Should the top one be tagged with aws-setup or aws-provision so that you can limit playbook execution?

mijicd committed:
Add osmetrics to cassandra cluster

Each cassandra node is provisioned with osmetrics. Updated the osmetrics
role to support both deb and rpm distributions.
